docs/_posts/2016-07-22-create-apps-with-no-configuration.md

```diff
@@ -25,7 +25,7 @@ Now you can use it to create a new app:
 create-react-app hello-world
 ```
 
-This will take a while as npm installs the transient dependencies, but once it’s done, you will see the list the commands you can run in the created folder:
+This will take a while as npm installs the transient dependencies, but once it’s done, you will see the list of the commands you can run in the created folder:
```

Comment:
Thanks! Let’s also make it “a list of commands” instead of “the list of the commands”?